bin/test

```diff
@@ -17,4 +17,14 @@ mix gen ./ kitchen_sink --ecto postgres --email --accounts --assets --asset-bund
 
 cd kitchen_sink && {
   bin/setup || { echo 'KitchenSink could not be set up!'; exit 1; }
+}
+
+# Generate a kitchen sink app and test it
+mix gen ./ kitchen_sink_travis --ecto postgres --email --accounts --assets --asset-bundler webpack --api graphql --html slim --ci travis --websockets --error-reporting honeybadger --deploy heroku --gettext || { echo 'KitchenSink app could not be generated'; exit 1; }
+
+cd kitchen_sink_travis && {
+  if [ ! -f .travis.yml ]; then
+    echo "Travis config file missing!"; exit 1; 
+  fi
+  bin/setup || { echo 'KitchenSink could not be set up!'; exit 1; }
```

Comment:
I don't think we need a repeat test just for the `travis` option, since it only adds a yaml file.